src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -16,7 +16,7 @@ module M = Messages
     - heterogeneous environments: https://link.springer.com/chapter/10.1007%2F978-3-030-17184-1_26 (Section 4.1) *)
 
 let widening_thresholds_apron = ResettableLazy.from_fun (fun () ->
-  let t = WideningThresholds.thresholds_incl_mul2 () in
+  let t = if GobConfig.get_bool "ana.autoselect" then WideningThresholds.ocatagon_thresholds () else WideningThresholds.thresholds_incl_mul2 () in
```

Comment:
The choice of which widening thresholds are used is currently tied together with `ana.autoselect`, but it might be better to have separate options for that, since we might also want to use the more optimized threshold collecting tactics outside of SV-COMP.